### ISSUE TITLE
Dark/light mode toggle switch has no focus indicator

### ISSUE THREAD

**Issue by danieldafoe** [mdn/yari#5439]
__Description__
When the dark/light mode toggle switch gets tab focus, there is no visible focus indicator.

__Proposed solution__
Apply either an `outline` or `box-shadow` to a parent element of the visually hidden `<input>` to indicate that the switch has focus and that any action taken will toggle its value.

**Comment by joelanman** [comment]
I think this is a duplicate of https://github.com/mdn/yari/issues/5420

**Comment by danieldafoe** [comment]
@joelanman Agreed. Seems this was already caught. Sorry about that!
